### PR TITLE
Improved neural network test

### PR DESCRIPTION
**Pull request by Andy-ch**
Minor improvements in tox.ini, requirements.txt and gen_one_symbol/main.py

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+pytest==3.0.7
```
Comment (ferc255) [inline requirements.txt]: Just wonder why to the top of the requirements? :D

**Comment by ferc255** [comment]
And yeah, probably close #15?

**Comment by esikachev** [comment]
@ferc255 should I close #15?

File: requirements.txt (hunk 1 above)
Comment (Andy-ch) [reply requirements.txt]: No real reason :) Just, like, it's the most important one for the tests :)

**Pull request merged**